tests/test_limiter.py

```diff
@@ -31,35 +31,29 @@ def test_general_functionality(rate, capacity):
     consume_one()
     time.sleep(float(capacity) / rate)
 
-    # NOTE(kgriffs): This works because we can consume at a much
-    #   higher rate relative to the replenishment rate, such that we
-    #   easily consume the total capacity before a single token can
-    #   be replenished.
```

Comment:
when we came to run these tests against our Django cache bucket storage we found that the `100, 100` parameter case failed

this is because the cache backend is slower than directly accessing a dict and it was enough that this approach didn't work any more with a high replenish rate

so here we update the test to make use of the actual elapsed time rather than a 'fudge factor' ...this test now works fine even with slow storage